### ISSUE TITLE
The pyvisa package is not available on Apple Silicon

### ISSUE THREAD

**Issue by davidfokkema** [conda-forge/pyvisa-feedstock#34]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

On Apple Silicon (M1 and M2 processors) pyvisa-py can not currently be installed:
```
$ conda create -n pythondaq python pyvisa-py -c conda-forge
Collecting package metadata (current_repodata.json): done
Solving environment: unsuccessful attempt using repodata from current_repodata.json, retrying with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: /
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

Package python conflicts for:
pyvisa-py -> python[version='>=3.6']
pyvisa-py -> importlib-metadata -> python[version='>=2|>=3.11,<3.12.0a0|>=3|>=3.10,<3.11.0a0|>=3.10,<3.11.0a0|>=3.8|>=3.9,<3.10.0a0|>=3.9,<3.10.0a0|>=3.8,<3.9.0a0|>=3.8,<3.9.0a0|>=3.7|>=3.5',build=*_cpython]
python
```
I could not understand the problem from the above output, _but_ pyvisa-py requires pyvisa, and pyvisa is not available:
```
$ conda search -c conda-forge pyvisa
Loading channels: done
No match found for: pyvisa. Search: *pyvisa*
```
It _is_ possible to install both pyvisa and pyvisa-py from PyPI using pip.

### Installed packages

```shell
# packages in environment at /opt/homebrew/Caskroom/miniconda/base/envs/pythondaq:
#
# Name                    Version                   Build  Channel
bzip2                     1.0.8                h3422bc3_4    conda-forge
ca-certificates           2023.7.22            hf0a4a13_0    conda-forge
libexpat                  2.5.0                hb7217d7_1    conda-forge
libffi                    3.4.2                h3422bc3_5    conda-forge
libsqlite                 3.43.0               hb31c410_0    conda-forge
libzlib                   1.2.13               h53f4e23_5    conda-forge
ncurses                   6.4                  h7ea286d_0    conda-forge
openssl                   3.1.3                h53f4e23_0    conda-forge
pip                       23.2.1             pyhd8ed1ab_0    conda-forge
python                    3.11.5          h47c9636_0_cpython    conda-forge
readline                  8.2                  h92ec313_1    conda-forge
setuptools                68.2.2             pyhd8ed1ab_0    conda-forge
tk                        8.6.12               he1e0b03_0    conda-forge
tzdata                    2023c                h71feb2d_0    conda-forge
wheel                     0.41.2             pyhd8ed1ab_0    conda-forge
xz                        5.2.6                h57fd34a_0    conda-forge
```

### Environment info

```shell
active environment : pythondaq
    active env location : /opt/homebrew/Caskroom/miniconda/base/envs/pythondaq
            shell level : 2
       user config file : /Users/david/.condarc
 populated config files : /Users/david/.condarc
          conda version : 23.7.4
    conda-build version : not installed
         python version : 3.10.13.final.0
       virtual packages : __archspec=1=arm64
                          __osx=13.5.2=0
                          __unix=0=0
       base environment : /opt/homebrew/Caskroom/miniconda/base  (writable)
      conda av data dir : /opt/homebrew/Caskroom/miniconda/base/etc/conda
  conda av metadata url : None
           channel URLs : https://repo.anaconda.com/pkgs/main/osx-arm64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-arm64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /opt/homebrew/Caskroom/miniconda/base/pkgs
                          /Users/david/.conda/pkgs
       envs directories : /opt/homebrew/Caskroom/miniconda/base/envs
                          /Users/david/.conda/envs
               platform : osx-arm64
             user-agent : conda/23.7.4 requests/2.31.0 CPython/3.10.13 Darwin/22.6.0 OSX/13.5.2
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```

**Comment by jjerphan** [comment]
Hi @davidfokkema,

This issue should be addressed by #38. :slightly_smiling_face:

**Comment by davidfokkema** [comment]
Yes, wonderful, thanks!!
